### DISCUSSION TITLE
Native Date picker is behaving weirdly.

### DISCUSSION THREAD

**Discussion by Officer-R** [react-hook-form/react-hook-form#5737] [Q&A]
Hi @bluebill1049, hope everything is good during this pandemic.

**Describing the question**

`<input type="date" {...register("date")} />` 

This above input with type date is behaving differently.
I am not sure if it is working properly in safari or in other browsers.

1. Safari ScreenShot:

 ![Screenshot 2021-06-25 at 2 36 06 PM](https://user-images.githubusercontent.com/37110560/123400256-2faae500-d5c3-11eb-8c18-995d7c16cc45.jpg)

2. Firefox Dev Edition: 

![Screenshot 2021-06-25 at 2 41 05 PM](https://user-images.githubusercontent.com/37110560/123400453-6680fb00-d5c3-11eb-8541-360f13ab2419.jpg)

3. Brave Browser: 

![Screenshot 2021-06-25 at 2 37 47 PM](https://user-images.githubusercontent.com/37110560/123400175-19048e00-d5c3-11eb-9862-783e56fba0c9.jpg)

4. Chrome: (For making things sure I installed chrome latest version)

![Screenshot 2021-06-25 at 2 49 17 PM](https://user-images.githubusercontent.com/37110560/123401633-9977be80-d5c4-11eb-8211-0d930bf76b92.jpg)

To reproduce this behaviour: 

open this [codesandbox](https://codesandbox.io/s/react-hook-form-basic-validation-forked-i8stq?file=/src/index.js:657-707) link in different browsers.

Hope this is a valid question.

**Comment by bluebill1049** [comment]
This is browser-level stuff, and not related to hook form.

**Reply by jack-hermanson** [reply]
The native datepicker works fine when I use `useState` and do everything manually. Only breaks when I use this library. :( It's a shame, because this is the only thing stopping me from adopting it in my projects.
